netbox/users/migrations/0002_squashed_0004.py

```diff
@@ -59,20 +59,4 @@ class Migration(migrations.Migration):
                 ('objects', django.contrib.auth.models.GroupManager()),
             ],
         ),
-        migrations.CreateModel(
```

Comment:
Removing this model leaves an orphaned ContentType. A custom field may reference this as its object type; it also might be referenced by plugins. I'm not sure offhand what the best strategy is here. Ideally I suppose we'd merge it into the new CT for the custom User model.

Reply:
That should be addressed in netbox/netbox/settings.py it removes and migrates the Content Type...